client/src/app/components/home/home.component.ts

```diff
@@ -41,4 +42,16 @@ export class HomeComponent implements OnInit{
   toggleMenu(): void {
     this.isMenuOpen = !this.isMenuOpen;
   }
+
+  vanillaTiltAnimation() {
```

Comment:
create a function for figure vanilla tlt for section 3 image div

Reply:
done. it will be update